src/games/strategy/triplea/ui/screen/UnitsDrawer.java

```diff
@@ -123,12 +126,12 @@ public void draw(final Rectangle bounds, final GameData data, final Graphics2D g
                 null);
           }
           break;
-        case NONE:
-          // This Method draws the unit Image
-          graphics.drawImage(img.get(), placementPoint.x - bounds.x, placementPoint.y - bounds.y, null);
-          break;
       }
     }
+    else{
+      // This Method draws the unit Image
```

Comment:
DRY violation. Refactor the existing usages to a helper method or a helper class. This exact comment appears on line 112, with the same call to drawImage.